eahub/feedback/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.16 on 2021-01-11 17:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = []
+
+    operations = [
+        migrations.CreateModel(
+            name="Feedback",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("email", models.CharField(blank=True, max_length=30)),
```

Comment:
We should add creation time as a field so that we know when the feedback was made